projects/project_ideas.md

```diff
@@ -0,0 +1,59 @@
+# CS 7290: Project Ideas
+**Prof. Robert Ness**
+
+## 1. State-of-the-art Causal Inference methods
+
+You are encouraged to explore Causal Inference literature and to explore the papers that suits your research interests. For reference, take a look at the proceedings of NeurIPS and UAI for last four years. The applications of these methods do not limit to AI and computer science. They expand to Epidemiology, Social sciences, Medicines, Statistics and Political sciences. You can pick the case studies that suit your interest.
+
+**Expectations**: Comprehensive literature survey and understanding of the problem, implementation of state-of-the-art methods, comparative analysis of similar methods and suggesting possible improvements.
```

Comment:
Let's focus on implementation.  A literature survey is too open a standard in my opinion.